projects/configs/tag-operations/rt-plan.yaml

```diff
@@ -0,0 +1,46 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+- name: Operators Name
+  group: 0x0008
+  element: 0x1070
+  op: keep
+- name: Referenced SOP Class UID
+  group: 0x0008
+  element: 0x1150
+  op: keep
+- name: Referenced SOP Instance UID
+  group: 0x0008
+  element: 0x1155
+  op: replace_UID
+- name: RT Plan Label
+  group: 0x300a
+  element: 0x0002
+  op: keep
+- name: RT Plan Date
+  group: 0x300a
+  element: 0x0006
+  op: keep
+- name: RT Plan Time
+  group: 0x300a
+  element: 0x0007
+  op: keep
```

Comment:
Our general default has been to replace all dates and times, but happy for project-specific overrides for this. Or do we think it'd never be useful without the dates and times?

Reply:
ah yep that sounds reasonable, I think defaulting to replacing them should be okay